doc/libvmem/libvmem.7.md

```diff
@@ -159,6 +159,13 @@ Otherwise, _UW(vmem_check_version) returns a static string describing the
 reason for failing the version check. The returned string must not be modified
 or freed.
 
+When _UW(vmem_check_version) is called with both *major_required* and
+*minor_required* equal 0, the return value is a static string
+containing the detailed information about the library version, source
+version, build type, etc.
+>NOTE: The application should not make any assumptions about the format
```

Comment:
Than this functionality is pointless because it cannot be used. I'd rather have few new ctl entry points for this purpose.